x-pack/platform/packages/private/ml/field_stats_flyout/options_list_with_stats/option_list_with_stats.tsx

```diff
@@ -117,6 +117,13 @@ export const OptionListWithFieldStats: FC<OptionListWithFieldStatsProps> = ({
             onChange={() => {}}
             value={value}
             aria-labelledby={titleId}
+            onKeyDown={(e) => {
+              if (e.key === 'Enter' || e.key === 'ArrowDown') {
+                e.preventDefault();
```

Comment:
I honestly don't like idea of adding e.preventDefault(); and  e.stopPropagation();

Reply:
have you set `closePopover` for `EiuInputPopover` as well?